internal/pkg/scaffold/cmd.go

```diff
@@ -49,6 +49,7 @@ import (
 
 	"{{ .Repo }}/pkg/apis"
 	"{{ .Repo }}/pkg/controller"
+	"{{ .Repo }}/version/version"
```

Comment:
```suggestion
	"{{ .Repo }}/version"
```

Whoops, that was my fault 😅

Reply:
Doh. My bad for not giving this PR a double check locally (aside from running `go test ./...`). I just copied my own project's change (poorly) into this PR.

Reply:
Local generation and usage looks to work:

```
go run cmd/manager/main.go
{"level":"info","ts":1569448514.0118022,"logger":"cmd","msg":"Version: 0.0.1"}
{"level":"info","ts":1569448514.011979,"logger":"cmd","msg":"Go Version: go1.13"}
{"level":"info","ts":1569448514.01199,"logger":"cmd","msg":"Go OS/Arch: darwin/amd64"}
{"level":"info","ts":1569448514.011998,"logger":"cmd","msg":"Version of operator-sdk: v0.10.0+git"}
```